lib/services/transaction-manager.js

```diff
@@ -0,0 +1,57 @@
+import Promise from 'bluebird'
+import * as constants from '../constants'
+
+export default class TransactionManager {
```

Comment:
Maybe a comment about what a transaction means in this case would be good.

Reply:
Documented here fef9183